examples/route53resolver/associateresolverrule/associateresolverrule.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: route53resolver.aws.crossplane.io/v1alpha1
+kind: AssociateResolverRule
```

```suggestion
kind: ResolverRuleAssociation
```
Could that be more inline with what exists?

---

i will set it to `RuleAssociation` then we have the same like provider-jet-aws

---

I'd lean towards how the AWS API names it in this repo, which is `ResolverRuleAssociation` I think looking at `GetResolverRuleAssociation` call.